File: FrontEnd(Angular 9)/src/app/_shared/model/compte.ts
export class  compte {
	id:number;
	solde:number;
	decouvert:number;
	client:string;

	getId():number{
        return this.id;
	}
	getSolde():number{
        return this.solde;
	}
	getDecouvert():number{
        return this.decouvert;
	}
	getclient():string{
        return this.client;
	}
	setId(id:number){
        this.id=id;
	}
	setSolde(solde:number){
        this.solde=solde;
	}
	setDecouvert(decouvert:number){
        this.decouvert=decouvert;
	}
	setClient(client:string){
        this.client=client;
    }
}

File: BackEnd(Java -jersy)/CompteBancaireWEBAPI/src/model/compte.java
package model;

public class compte {
	private int id;
	private double solde;
	private double decouvert;
	private String client;
	
	
	public compte() {
	}

	
	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public double getSolde() {
		return solde;
	}

	public void setSolde(double solde) {
		this.solde = solde;
	}

	public double getDecouvert() {
		return decouvert;
	}

	public void setDecouvert(double decouvert) {
		this.decouvert = decouvert;
	}


	public String getClient() {
		return client;
	}


	public void setClient(String client) {
		this.client = client;
	}

	
	
}
